templates/taxi/car_detail.html

```diff
@@ -1,12 +1,19 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.first_name }} {{ driver.last_name }}</li>
-  {% endfor %}
-  </ul>
+  <h1 class="taxi-title">{{ car.manufacturer.name }} {{ car.model }}</h1>
+  {% if car.drivers %}
+    <h2 class="taxi-title">List of drivers:</h2>
+    <ul class="driver-list">
+      {% for driver in car.drivers.all %}
```

Comment:
Verify that `car.drivers` is a related manager that supports the `.all()` method. If not, this could lead to errors in your application.